tests/test_mpscircuit.py

```diff
@@ -215,20 +250,96 @@ def do_test_proj(test_circucits, external_wavefunction):
     np.testing.assert_allclose(proj, proj_ref, atol=1e-12)
 
 
+def do_test_tensor_input(test_circuits: type_test_circuits):
+    (
+        c,
+        w_c,
+        mps,
+        w_mps,
+        mps_exact,
+        w_mps_exact,
+    ) = test_circuits
+    newmps = tc.MPSCircuit(
+        mps._nqubits,
+        tensors=mps.get_tensors(),
+        center_position=mps.get_center_position(),
+    )
+    for t1, t2 in zip(newmps.get_tensors(), mps.get_tensors()):
+        np.testing.assert_allclose(
+            tc.backend.numpy(t1), tc.backend.numpy(t2), atol=1e-12
+        )
+
+
+def do_test_measure(test_circuits: type_test_circuits):
+    (
+        c,
+        w_c,
+        mps,
+        w_mps,
+        mps_exact,
+        w_mps_exact,
+    ) = test_circuits
+    index = [6, 5, 2, 9]
+    status = tc.backend.convert_to_tensor([0.1, 0.3, 0.5, 0.7])
+    result_c = c.measure(*index, with_prob=True, status=status)
+    result_mps = mps.measure(*index, with_prob=True, status=status)
+    result_mps_exact = mps_exact.measure(*index, with_prob=True, status=status)
+    np.testing.assert_allclose(result_mps[0], result_c[0], atol=1e-8)
```

Comment:
why the measure returns the same result? shouldn't it be stochastic?

Reply:
Because a predefined status is used as input?

Reply:
sorry, my bad. not see the status argument